src/main/java/alternate/current/mixin/WorldChunkMixin.java

```diff
@@ -97,7 +97,7 @@ public WireNode setWire(WireNode wire) {
 		return ((IChunkSection)section).setWire(wire);
 	}
 	
-	private ChunkSection getSection(int y) {
+	public ChunkSection getSection(int y) {
```

Comment:
Why should this method be public? It can't be accessed outside this class without an interface anyway.